packages/flutter/lib/src/widgets/pop_scope.dart

```diff
@@ -10,6 +10,12 @@ import 'routes.dart';
 
 /// Manages system back gestures.
 ///
+/// A system back gesture is a swipe inward from near the edge of the screen on
+/// Android. It is recognized by Android before being passed to Flutter. iOS has
+/// a similar gesture, but it is recognized in Flutter by
```

Comment:
Can you use this widget to control the behavior of CupertinoRouteTransitionMixin then?

Reply:
Setting `canPop: false` will disable the CupertinoRouteTransitionMixin  back gesture and its animation altogether. I'll explain this better.